tests/libxc/dft-water/input.dat

```diff
@@ -0,0 +1,152 @@
+ 'revTPSSh':    -76.353009954,        #TEST
+      'BOP':   -76.3655205429,        #TEST
+    'B1B95':   -76.3542909285,        #TEST
+  'HCTH147':   -76.3795700801,        #TEST
+    'M11-L':   -76.3634719346,        #TEST
+   'M08-SO':   -76.3328381854,        #TEST
+     'PBE0':   -76.3011288827,        #TEST
+      'PBE':   -76.2983301284,        #TEST
+     'PW91':   -76.3556764834,        #TEST
+    'B97-D':   -76.3448936668,        #TEST
+    'MPW1K':   -76.3571352506,        #TEST
+   'MPWB1K':   -76.3458871109,        #TEST
+  'HCTH407':   -76.3751427842,        #TEST
+     'dlDF':   -76.8428067094,        #TEST
+   'M08-HX':    -76.349820226,        #TEST
+  'HCTH120':   -76.3756781889,        #TEST
+   'M05-2X':   -76.3685940515,        #TEST
+   'B3PW91':   -76.3574933252,        #TEST
+  'wB97X-V':   -76.3540786402,        #TEST
+     'EDF1':   -76.3971372179,        #TEST
+  'revTPSS':   -76.3563235056,        #TEST
+  'wB97X-D':   -76.3625422648,        #TEST
+    'B97-3':    -76.365757473,        #TEST
+ 'MGGA_MS1':   -76.3863611957,        #TEST
+      'M11':   -76.3548246737,        #TEST
+   'PW6B95':   -76.4636801397,        #TEST
+   'B3LYP5':   -76.3479565264,        #TEST
+    'X3LYP':   -76.3550695373,        #TEST
+     'BLYP':   -76.3649513839,        #TEST
+   'HCTH93':   -76.3690959506,        #TEST
+   'M06-HF':   -76.3438580561         #TEST
+#   'PBEOP':   -76.3113060897,        #TEST
+# 'MN12-SX':   -76.3292426233,        #TEST
+#     'GAM':    -76.351646627,        #TEST
+#'MGGA_MS2h':  -76.3874519076,        #TEST
+# 'SOGGA11':   -76.3728240224,        #TEST
+# 'LRC-BOP':   -76.2298882864,        #TEST
+#    'PKZB':   -76.2392402265,        #TEST
+# 'revPBE0':   -76.3469704408,        #TEST
+#  'N12-SX':   -76.3362254975,        #TEST
+#  'revPBE':   -76.3594661693,        #TEST
+# 't-HCTHh':   -76.3688723507,        #TEST
+# 'wM06-D3':   -76.3597621058,        #TEST
+#   'PBE50':   -76.3045269871,        #TEST
+#     'N12':   -76.3313530822,        #TEST
+#   'M06-L':      -76.3773597,        #TEST
+#  'MN12-L':   -76.3121263959,        #TEST
+#  'B3TLAP':   -76.1762459451,        #TEST
+# 'MPW1LYP':   -76.3515538422,        #TEST
+#'SOGGA11-X':  -76.3579662706,        #TEST
+#    'VSXC':   -76.4031806261,        #TEST
+# 'MPW1PBE':   -76.3399258699,        #TEST
+#   'SOGGA':   -75.9775522776,        #TEST
+#    'SCAN':   -76.3525050753,        #TEST
+#'MPW1PW91':   -76.3639997875,        #TEST
+#     'BMK':   -76.3477995384,        #TEST
+#  'MN15-L':   -76.3003777073,        #TEST
+#  'PBEsol':   -76.0376457772,        #TEST
+#   'SCAN0':   -76.3393392953,        #TEST
+#'MGGA_MVS':   -76.3432912834,        #TEST
+#'MGGA_MVSh':  -76.3375136868,        #TEST
+#  'wM05-D':    -76.355280694,        #TEST
+#  't-HCTH':   -76.3695334542,        #TEST
+# 'BP86VWN':   -76.3897099046,        #TEST
+}                                     #TEST
+
+old_psi4_data = {                     #TEST 
```

Comment:
throw in a comment about the version. If it's the one I sent you, its ~April 2017 and you can get a sha from the output file.